test/controllers/term_controller_test.exs

```diff
@@ -117,42 +136,55 @@ defmodule CoursePlanner.TermControllerTest do
     assert html_response(conn, 404)
   end
 
-  test "doesn't show deleted term", %{conn: conn} do
```

Comment:
ok I checked. since the status is removed "doesn't show inexisting term" test already test the deleted scenario. here is the code
`test "doesn't show inexisting term", %{conn: conn} do
     conn = get conn, term_path(conn, :show, 1)
      assert html_response(conn, 404)
    end`